### DISCUSSION TITLE
EDC version in DataDashboard vs Connectors

### DISCUSSION THREAD

**Discussion by JorgeBarto** [eclipse-dataspaceconnector/DataDashboard#254] [General]
Hi everyone!
first of all, congrats for your work.
I'm new at all of this and I would like to make myself clear about it.
I was able to clone and run DataDashboard project and therefore to see DataDashboard UI in order to manage assets, policies etc. What It seems odd to me is that the file named libs.versions.toml in the DataDashboard project has EDC version 0.2.0 while Connectors project has 0.8.1 EDC version. So the question could be, are both projects compatible with each other?
Thanks in advantage!

**Comment by Moritz72** [comment]
They are currently not compatible as far as I can tell. If you manually update the version of [edc-connector-client](https://github.com/Think-iT-Labs/edc-connector-client) and do some minor tweaks the Dashboard should work when running a separate EDC on version 0.8.1. With that I got it to work for EDC version 0.5.1 at least.

**Comment by JorgeBarto** [comment]
Hi again!
I just clone the repo [DataDashboard](https://github.com/eclipse-edc/DataDashboard) and run a Docker-compose-Up. Everything looks fine to me (2 connector and their respective Dashboards are running). 
I'm also able to access Data Dashboard UI but, while accessing any option menu such as: asset, policies, contract definitions... the error seems always similar and I can not see any form.
http://localhost:28080/management/v3/assets/request net::**ERR_CONNECTION_REFUSED**
http://localhost:28080/management/v2/policydefinitions/request net::**ERR_CONNECTION_REFUSED**
I left everything by default after the repo was cloned. Should I made any changes? Could be something wrong on my side/server?
Thanks!!

**Reply by Moritz72** [reply]
Looking at the doocker-compose.yml I'm not quite sure why. But everything works for me if I just run

```
docker compose up
```

The Connectors Management APIs are reachable under http://localhost:18080/management and http://localhost:28080/management. But for you this does not seem to be the case.

Can you check whether you get anything for http://localhost:19193/management/v3/assets (should be HTTP ERROR 405 Method Not Allowed). If so you can change the two json files in deployment/conf: 19193 instead of 18080 and 29193 instead of 28080.

But coming back to your original question, keep in mind that those are EDCs running on version 0.2.0 and newer versions will mostly not work with this current Dashboard.
